lib/util/template/index.spec.ts

```diff
@@ -423,4 +423,104 @@ describe('util/template/index', () => {
       expect(output).toBe('bar');
     });
   });
+
+  describe('lookupArray', () => {
+    it('performs lookup for every array element', () => {
+      const output = template.compile(
+        '{{#each (lookupArray upgrades "prBodyDefinitions")}} {{{Issue}}}{{/each}}',
+        {
+          upgrades: [
+            {
+              prBodyDefinitions: {
+                Issue: 'ABC-123',
+              },
+            },
+            {},
+            {
+              prBodyDefinitions: {
+                Issue: 'DEF-456',
+              },
+            },
+            null,
+            undefined,
+          ],
+        },
+      );
+
+      expect(output).toBe(' ABC-123 DEF-456');
+    });
+
+    it('handles null input array', () => {
+      const output = template.compile(
+        '{{#each (lookupArray testArray "prBodyDefinitions")}} {{{Issue}}}{{/each}}',
+        {
+          testArray: null,
+        },
+        false,
+      );
+
+      expect(output).toBe('');
+    });
+
+    it('handles empty string key', () => {
+      const output = template.compile(
+        '{{#each (lookupArray testArray "")}} {{{.}}}{{/each}}',
+        {
+          testArray: [
+            {
+              '': 'ABC-123',
+            },
+          ],
+        },
+        false,
+      );
+
+      expect(output).toBe(' ABC-123');
+    });
+
+    it('handles null key', () => {
```

Comment:
test null object

Reply:
There is already "handles null input array" test, isn't it doing what you're asking?

And "performs lookup for every array element" test already tests null elements.

Reply:
ok